.github/CODEOWNERS

```diff
@@ -92,6 +92,8 @@ extension/awsproxy/                                                     @open-te
 extension/basicauthextension/                                           @open-telemetry/collector-contrib-approvers @jpkrohling @svrakitin @frzifus
 extension/bearertokenauthextension/                                     @open-telemetry/collector-contrib-approvers @jpkrohling @pavankrish123 @frzifus
 extension/encodingextension/                                            @open-telemetry/collector-contrib-approvers @MovieStoreGuy
+extension/encodingextension/jaeger/                                     @open-telemetry/collector-contrib-approvers @MovieStoreGuy
+extension/encodingextension/text/                                       @open-telemetry/collector-contrib-approvers @MovieStoreGuy
```

Comment:
Why embedding? We don't follow this pattern in other components. I think it should be 

`extension/jaegerencodingextension` and `extension/textencodingextension`

2 more Q: 

1. Where do you want to use the text encoding extension?
2. Why are you not adding yourself as the codeowner? :)

Reply:
I'll remove the embedding. We do embed for storage but it's fine, we don't need to do that.

1. We use the `text` encoding for kafka/google pubsub/azure eventhub. We also use a bytes encoding. I know.
2. You really think I'm not going to jump at the opportunity to leave @MovieStoreGuy with the bag? If you insist I can add myself.

Reply:
> I'll remove the embedding. We do embed for storage but it's fine, we don't need to do that.

Oh, you're right. I was wrong. 

What about `extension/encoding/jaegerencongingextension`? I think it's important to have the `extension` part in the module/package name. Not sure why storage extension doesn't do that

Reply:
> You really think I'm not going to jump at the opportunity to leave @MovieStoreGuy with the bag? If you insist I can add myself.

Please add yourself. I thing @MovieStoreGuy would agree :)

Reply:
OK no worries. Thanks for the quick feedback. I should have this done by tonight, just shuffling things around.

Reply:
Pushed with your recommendations.